embedded-graphics/src/primitives/polyline/mod.rs

```diff
@@ -122,13 +122,15 @@ impl<'a> Transform for Polyline<'a> {
     ///
     /// let polyline = Polyline::new(&points);
     /// let moved = polyline.translate(Point::new(10, 12));
+    /// let moved2 = moved.translate(Point::new(10, 12));
```

Comment:
This shouldn't clutter the usage example IMO. Could you move this to a separate test?

Reply:
Of course, I can do anything :P